sopel/loader.py

```diff
@@ -140,7 +120,7 @@ def clean_callable(func, config):
     prefix = config.core.prefix
     help_prefix = config.core.help_prefix
     func._docs = {}
-    doc = trim_docstring(func.__doc__)
+    doc = inspect.cleandoc(func.__doc__ or '').splitlines()
```

Comment:
I appreciate what this replacement (and associated function deletion) is doing, but I don't think it belongs in this PR. Would you mind splitting it out? :smile_cat:

Unlike the added tests, this change really requires messing with the resulting output interactively via the `help` module to see if it affects anything. It shouldn't, but… I'm careful.